### PR TITLE
AnP: activate config file cli

### PR DESCRIPTION
**Pull request by ebelter**

### CODE REVIEW

File: lib/perl/Genome/Config/AnalysisProject/Command/ActivateConfigFile.pm [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+use warnings;
+
+use Genome;
+
+class Genome::Config::AnalysisProject::Command::ActivateConfigFile {
+    is => 'Command::V2',
+    has_input => [
+        profile_item => {
+            is => 'Genome::Config::Profile::Item',
+            doc => 'Specified config file will no longer be used',
```
Comment (tmooney) [inline lib/perl/Genome/Config/AnalysisProject/Command/ActivateConfigFile.pm]: This documentation is not accurate.

**Comment by tmooney** [comment]
This brings up the same issues as #1151.

**Comment by jasonwalker80** [comment]
I would reiterate that activating a disabled config should not be allowed.  Is there a use case for activating a config file?  Why was it inactivated or disabled in the first place?

**Comment by ebelter** [comment]
My use case involved reactivating a config that had a processing profile that skipped digital expression. I added a config to reprocess some instrument data that failed with HT Seq. After reprocessing, I inactivated the config file.

I thought about not allowing disabled configs to be activated, and should be restricted.

Additionally, the file could just be added back to the AnP every time, even a disabled one.

**Comment by ebelter** [comment]
Jason and I have talked out a different solution. PR forthcoming.

**Pull request closed without merging**